### PR TITLE
[stable/cstor]: update charts to 2.6.0

### PR DESCRIPTION
**Pull request by akhilerm**
- update chart version
- update dependent chart version
- update README
- update values.yaml

Signed-off-by: prateekpandey14 <prateek.pandey@mayadata.io>
Signed-off-by: Akhil Mohan <akhil.mohan@mayadata.io>

Dependent on openebs/node-disk-manager#540

### CODE REVIEW

File: deploy/helm/charts/README.md [hunk 1]
```diff
@@ -81,54 +81,54 @@ The following table lists the configurable parameters of the OpenEBS CStor chart
...
+| csiController.attacher.image.repository | string | `"sig-storage/csi-attacher"` |  CSI attacher image repo |
 | csiController.attacher.image.tag | string | `"v3.1.0"` | CSI attacher image tag |
 | csiController.attacher.name | string | `"csi-attacher"` |  CSI attacher container name|
 | csiController.componentName | string | `""` | CSI controller component name |
 | csiController.driverRegistrar.image.pullPolicy | string | `"IfNotPresent"` | CSI driver registrar image pull policy  |
-| csiController.driverRegistrar.image.registry | string | `"quay.io/"` | CSI driver registrar image registry |
-| csiController.driverRegistrar.image.repository | string | `"k8scsi/csi-cluster-driver-registrar"` | CSI driver registrar image repo |
+| csiController.driverRegistrar.image.registry | string | `"k8s.gcr.io/"` | CSI driver registrar image registry |
+| csiController.driverRegistrar.image.repository | string | `"sig-storage/csi-cluster-driver-registrar"` | CSI driver registrar image repo |
 | csiController.driverRegistrar.image.tag | string | `"v1.0.1"` |  CSI driver registrar image tag|
```
Comment (shubham14bajpai) [inline deploy/helm/charts/README.md:100]: Can we change this corresponding to the values.yaml
```suggestion
| csiController.driverRegistrar.image.tag | string | `"v2.1.0"` |  CSI driver registrar image tag|
```

File: deploy/helm/charts/README.md [hunk 2]
```diff
@@ -81,54 +81,54 @@ The following table lists the configurable parameters of the OpenEBS CStor chart
...
 | csiController.snapshotter.name | string | `"csi-snapshotter"` | CSI snapshotter container name |
 | csiController.tolerations | list | `[]` | CSI controller pod tolerations |
 | csiNode.annotations | object | `{}` | CSI Node annotations |
 | csiNode.componentName | string | `"openebs-cstor-csi-node"` | CSI Node component name |
 | csiNode.driverRegistrar.image.pullPolicy | string | `"IfNotPresent"` | CSI Node driver registrar image pull policy|
-| csiNode.driverRegistrar.image.registry | string | `"quay.io/"` | CSI Node driver registrar image registry |
-| csiNode.driverRegistrar.image.repository | string | `"k8scsi/csi-node-driver-registrar"` | CSI Node driver registrar image repository |
+| csiNode.driverRegistrar.image.registry | string | `"k8s.gcr.io/"` | CSI Node driver registrar image registry |
+| csiNode.driverRegistrar.image.repository | string | `"sig-storage/csi-node-driver-registrar"` | CSI Node driver registrar image repository |
 | csiNode.driverRegistrar.image.tag | string | `"v1.0.1"` |  CSI Node driver registrar image tag|
```
Comment (shubham14bajpai) [inline deploy/helm/charts/README.md:132]: Can we change this corresponding to the values.yaml
```suggestion
| csiController.driverRegistrar.image.tag | string | `"v2.1.0"` |  CSI driver registrar image tag|
```

File: deploy/helm/charts/README.md (hunk 1 above)
Comment (akhilerm) [reply deploy/helm/charts/README.md]: Done.

**Review by shubham14bajpai — APPROVED** [review]

**Review by prateekpandey14 — COMMENTED** [review]
@akhilerm we have to update `UpgradeTasks` and `CstorVolumeAttachement` CRDs as well.

**Comment by akhilerm** [comment]
Done. @prateekpandey14

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Pull request merged**